src/hardware/link-mso19/protocol.c

```diff
@@ -24,38 +25,34 @@
 
 /* serial protocol */
 #define mso_trans(a, v) \
-	(((v) & 0x3f) | (((v) & 0xc0) << 6) | (((a) & 0xf) << 8) | \
+	g_htons(((v) & 0x3f) | (((v) & 0xc0) << 6) | (((a) & 0xf) << 8) | \
 	((~(v) & 0x20) << 1) | ((~(v) & 0x80) << 7))
 
 static const char mso_head[] = { 0x40, 0x4c, 0x44, 0x53, 0x7e };
 static const char mso_foot[] = { 0x7e };
 
-SR_PRIV int mso_send_control_message(struct sr_serial_dev_inst *serial,
-				     uint16_t payload[], int n)
+static int mso_send_control_message(struct sr_serial_dev_inst *serial,
+				    uint16_t payload[], int n)
 {
-	int i, w, ret, s = n * 2 + sizeof(mso_head) + sizeof(mso_foot);
+	int w, ret;
```

Comment:
I'd prefer more  speaking variable names

Reply:
I didn't write this code, so I left the variable names as they were originally written.